src/test/java/com/snowflake/kafka/connector/internal/streaming/StreamingClientHandlerTest.java

```diff
@@ -63,7 +63,7 @@ public void testCreateClientException() {
   @Test
   public void testCreateClientOverrideBdecVersion() {
```

Comment:
remove this test?

Reply:
I think we still need this in the future since we need a new BDEC version for supporting Iceberg or anything in case we need to update the format

Reply:
Ack